src/base1/test-dbus.js

```diff
@@ -130,6 +130,22 @@ QUnit.asyncTest("owned messages", function() {
     acquire_name();
 });
 
+QUnit.asyncTest("call method on non-existent name", function() {
+    assert.expect(1);
+
+    var dbus = cockpit.dbus("com.example.nonexistent");
+    dbus.call("/foo", "com.example.nonexistent", "Foo", []).
+        then(function(reply) {
+           assert.ok(false, "should not return a reply");
+        }).
+        catch(function(error) {
+            assert.equal(error.name, "org.freedesktop.DBus.Error.ServiceUnknown");
+        }).
+        always(function() {
+            QUnit.start();
+        });
+});
+
```

Comment:
Another test may need changes here:

```
FAIL: dist/base1/test-dbus.html 213 - expected: "org.freedesktop.DBus.Error.ServiceUnknown", got: null, test: call method on non-existent name, module: , source: at Object.equal (http://localhost:59035/dist/base1/qunit.js:2194:12)
```

Reply:
Ah, I forgot `{ bus: "session" }` in the test. Thanks.